tests/w2w_test.py

```diff
@@ -103,13 +161,50 @@ def test_create_wrf_gridinfo(tmpdir):
     }
     create_wrf_gridinfo(info)
     assert os.listdir(tmpdir) == ['dst_gridinfo.tif']
-    tif = rioxarray.open_rasterio(info['dst_gridinfo'])
+    tif = rxr.open_rasterio(info['dst_gridinfo'])
     assert tif.rio.crs.to_proj4() == '+init=epsg:4326'
     assert tif.rio.transform() == Affine(
             0.01000213623046875, 0.0, -1.4050254821777344,
             0.0, 0.010000228881835938, 41.480000495910645,
     )
 
+def test_get_SW_BW(capsys):
+
+    ucp_table = pd.read_csv('testing/LCZ_UCP_lookup.csv',
+        sep=',', index_col=0
+    ).iloc[:17, :]
+    SW, BW = _get_SW_BW(ucp_table)
+
+    assert list(SW[:10]) == [
+        20.0,
+        14.0,
+        5.2,
+        50.0,
+        35.0,
+        13.0,
+        3.3333333333333335,
+        32.5,
+        43.333333333333336,
+        28.571428571428573
+    ]
+    assert list(BW[:10]) == [
+        22.22222222222222,
+        22.000000000000004,
+        9.533333333333337,
+        42.85714285714285,
+        26.25,
+        13.0,
+        25.000000000000007,
+        28.888888888888893,
+        43.333333333333336,
+        23.80952380952381
+    ]
```

Comment:
These are floating point issues. Pytest has the nice thing called `pytest.approx` (https://docs.pytest.org/en/6.2.x/reference.html#pytest-approx), where you can sepcify a tolerance, so e.g. `1e-6`. This should also work on lists